internal/api/grpc/admin/integration_test/restrictions_allowed_languages_test.go

```diff
@@ -51,7 +51,7 @@ func TestServer_Restrictions_AllowedLanguages(t *testing.T) {
 			require.Equal(ttt, language.Make(defaultLang.Language), language.English)
 		})
 		tt.Run("the discovery endpoint returns all supported languages", func(ttt *testing.T) {
-			awaitDiscoveryEndpoint(ttt, instance.Domain, supportedLanguagesStr, nil)
+			awaitDiscoveryEndpoint(ttt, iamOwnerCtx, instance.Domain, supportedLanguagesStr, nil)
```

Comment:
i know the iamOwnerCtx works, put makes the impression an authorization is needed
(please see also the other changes)

Reply:
True there was a context.Background used in the underlying function, will changed that again.

Reply:
i probably would have used the `ctx`, but context.Background() works as well
but there are still some `iamOwnerCtx` (line 95 and 98)